sections/introduction.tex

```diff
@@ -1,9 +1,15 @@
 \chapter{Introduction}\label{ch:introduction}
-Here is the introduction. The next chapter is chapter~\ref{ch:ch2label}.
-
-
-a new paragraph
-
+Recommender systems(RS) are widely used to combat information overload for users when accessing new information by recommending specific items that the user is likely to find compatible \cite{YouTubeNeural,IndustryPerspective}.
+RS are conventionally defined through three categories - \textit{content-based}, \textit{collaborative filtering} and \textit{hybrid}.
+\textit{Content-based} RS provide recommendations based on content information such as genre, \textit{collaborative filtering} recommend items based on similar users, and \textit{hybrid} combines the approaches\cite{ContextSurvey2020}.
+\textit{Context-Aware Recommender Systems} (CARS) exist as a subset of RS, aiming to incorporate contextual information into the process of recommendation.
+For a movie RS, the objective is to estimate a rating function $R: User \times Item \rightarrow Rating$ for the $(user, \ item)$ pairs that are not yet rated\cite[p.~191-227]{RecommenderHandbook2015}.
```

Comment:
```suggestion
Contextual information is intuitively useful for making recommendations, in a movie recommendation situation this context could be company, time, or mood.
For example, a user may not want to watch the same movies in the company of their significant other, as they would with their parents.
Likewise, time could be an important factor, since movies such as \textit{A Christmas Story} may be more relevant during the winter months than during summer.\\\
Generally, for a movie RS, the objective is to estimate a rating function $R: User \times Item \rightarrow Rating$ for the $(user, \ item)$ pairs that are not yet rated \cite[p.~191-227]{RecommenderHandbook2015}.
```